src/zcl_aff_writer_xslt.clas.testclasses.abap

```diff
@@ -942,10 +942,11 @@ CLASS ltcl_integration_test IMPLEMENTATION.
     json_writer->if_sxml_writer~set_option( option = if_sxml_writer=>co_opt_indent value = '2' ).
     CALL TRANSFORMATION (st_name) SOURCE root = test_type RESULT XML json_writer.
 
-
-    cl_aff_content_handler_factory=>get_handler_for_plain_text( )->deserialize(
```

Comment:
The cl_aff_content_handler_factory is also called in an other place of this class. So it should be removed there, too.